src/api/ArbeidsgiverMapper.ts

```diff
@@ -1,6 +1,6 @@
 import { Organisasjon } from '@navikt/bedriftsmeny/lib/organisasjon';
 
-export const convertResponseDataToOrganisasjon = (data): Organisasjon[] => data.map(organisasjon => ({
+export const mapArbeidsgiver = (data): Organisasjon[] => data.map(organisasjon => ({
```

Comment:
Hvis du bruker export default så slipper du med import mapArbeidsgiver from .....

Men det er lurt å ikke eksportere en anonym funksjon for å gjøre det lettere å debugge.